docs/supervisor-datamgr.md

```diff
@@ -0,0 +1,72 @@
+# Data manager configuration for vSphere with Kubernetes
+In a vSphere with Kubernetes environment, the data manager should be installed as a Virtual Machine on vSphere. It does not come
+up as container, as it does in a kubernetes vanilla vSphere environment. This is due to networking limitation in accessing
+vCenter/management network from TKGS clusters or PodVMs.
+
+This document discusses the installation procedure for backup data manager for velero plugin for vSphere with kubernetes.
+
+## Changes with release 1.1.0
+Support of vSphere with kubernetes (supervisor/workload cluster and TKGS cluster) is being added with the release 1.1.0 release.
```

Comment:
We should probably add a table to list all the terms.

Reply:
Would this be there in the main documentation? Lets add it once all the documentation has been added.

Reply:
Sure.